bumble/rfcomm.py

```diff
@@ -154,15 +152,54 @@ def make_service_sdp_records(
 
     if uuid:
         records.append(
-            ServiceAttribute(
-                SDP_SERVICE_CLASS_ID_LIST_ATTRIBUTE_ID,
-                DataElement.sequence([DataElement.uuid(uuid)]),
+            sdp.ServiceAttribute(
+                sdp.SDP_SERVICE_CLASS_ID_LIST_ATTRIBUTE_ID,
+                sdp.DataElement.sequence([sdp.DataElement.uuid(uuid)]),
             )
         )
 
     return records
 
 
+# -----------------------------------------------------------------------------
+async def search_channels_from_sdp(connection: Connection) -> Dict[int, List[UUID]]:
```

Comment:
suggest: maybe this could be named `find_rfcomm_channels ` (see other comment for alignement with similar functionality currently in `bench.py`)

Reply:
Done